src/fides/api/schemas/privacy_preference.py

```diff
@@ -27,16 +76,47 @@ class ConsentOptionCreate(FidesSchema):
 
 
 class PrivacyPreferencesRequest(FidesSchema):
-    """Request body for creating PrivacyPreferences."""
+    """Request body for creating PrivacyPreferences.
+
+
+    "preferences" key reserved for saving preferences against a privacy notice.
+
+    New *_preferences fields are used for saving preferences against various tcf components.
+    """
 
     browser_identity: Identity
     code: Optional[SafeStr]
-    preferences: conlist(ConsentOptionCreate, max_items=50)  # type: ignore
+    preferences: conlist(ConsentOptionCreate, max_items=200) = []  # type: ignore
+    purpose_preferences: conlist(TCFPurposeSave, max_items=200) = []  # type: ignore
+    special_purpose_preferences: conlist(TCFSpecialPurposeSave, max_items=200) = []  # type: ignore
+    vendor_preferences: conlist(TCFVendorSave, max_items=200) = []  # type: ignore
+    feature_preferences: conlist(TCFFeatureSave, max_items=200) = []  # type: ignore
+    special_feature_preferences: conlist(TCFSpecialFeatureSave, max_items=200) = []  # type: ignore
+    system_preferences: conlist(TCFVendorSave, max_items=200) = []  # type: ignore
```

Comment:
Updated request body for saving preferences.  You can send in a list of preferences for various privacy notices per usual, but now you can send in preferences against multiple TCF components simultaneously.